doc/support.rst

```diff
@@ -119,13 +116,13 @@ Then you can build the documentation from within the ``doc/`` directory as part
 .. note::
 
     Documentation as part of the package should be largely limited to the API.
-    More elaborate documentation on how to integrate **signac-flow** into a computational workflow should be documented as part of the `framework documentation <https://docs.signac.io>`_, which is maintained `here <https://bitbucket.org/glotzer/signac-docs>`_.
+    More elaborate documentation on how to integrate **signac-flow** into a computational workflow should be documented as part of the `framework documentation <https://docs.signac.io>`_, which is maintained `here <https://github.com/glotzerlab/signac-docs>`_.
 
 
 Updating the changelog
 ----------------------
 
-To update the changelog, add a one-line description to the `changelog.txt <https://bitbucket.org/glotzer/signac-flow/src/master/changelog.txt>`_ file within the ``next`` section.
+To update the changelog, add a one-line description to the `changelog.txt <https://github.com/glotzerlab/signac-flow/blob/master/changelog.txt>`_ file within the ``next`` section.
```

Comment:
I've updated all changelogs links to point directly to the changelog on docs.signac.io/... .

Reply:
@csadorf I'll change this, but the link I see in [doc/support.rst](https://github.com/glotzerlab/signac/blob/develop/doc/support.rst) for signac on develop still links to GitHub, unless I'm looking in completely the wrong place.